<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiDialogCommandEditor</class>
 <widget class="QDialog" name="uiDialogCommandEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command Editor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_pButtonHelp">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox1">
     <layout class="QGridLayout">
      <item row="1" column="0" colspan="6">
       <widget class="QTextEdit" name="m_pTextEditComment" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>32767</width>
          <height>90</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="m_pButtonAddCommand">
        <property name="text">
         <string>Add command</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="m_pButtonDeleteCommand">
        <property name="text">
         <string>Delete command</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="6">
       <widget class="QTabWidget" name="m_pTabWidget">
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>cmd 1</string>
         </attribute>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <widget class="QCheckBox" name="m_pCheckDefault">
            <property name="text">
             <string>Use as default command</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel2">
            <property name="text">
             <string>&lt;p align=&quot;center&quot;&gt;Command Line&lt;/p&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QTextEdit" name="m_pTextEditCommand" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" stdset="0">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" colspan="6">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>&lt;p align=&quot;center&quot;&gt;Explaining header&lt;/p&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QComboBox" name="m_pComboVar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>TEMP_PATH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PROJECT_PATH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SOUND_FILES</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WIDTH</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HEIGHT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FORMAT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DVDAUTHOR_XML</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MENU_NAME</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BACKGROUND_FILE_NAME</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BACKGROUND_FULL_NAME</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>START_POS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DURATION</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="textLabel1_2_2">
        <property name="text">
         <string>var</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="spacer2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="m_pButtonAddToCommand">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QTextEdit</class>
   <extends>QWidget</extends>
   <header>QTextEdit</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>uiDialogCommandEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>uiDialogCommandEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
